src/pages/examples/_config.js

```diff
@@ -70,6 +70,16 @@ export const examples = [
         updatedAt: "2022-01-31",
         tech: ["angular", "rxjs"],
     },
+    {
+        title: "Todo App (React)",
```

Comment:
Го поднимем выше к основной туду-аппе реактовской чтоб не потерялось

И чуть уточним название (т.к. это в метрики летит)

```suggestion
        title: "Todo App (React+Redux)",
```

Reply:
@azinit а то что при этом в дескрипшне будет дублироваться _(React version)_ - это ок?

Reply:
@EliseyMartynov  там же будет `QuickStart todo-app example for React developers (Redux version)` не?

Reply:
В метрики это конечно не летит, но по дескрипшну туду-аппы должны отличаться
(нам бы кста туду-аппу на эффекторе тоже бы подредачить, ща скину скрин)

Reply:
Тип раз уж у нас не просто "реакт аппы" а еще разделенные по стеку (после добавления редакс) я бы уточнил в конфиге первой тудуаппы - что она на эффекторе (поправить тайтл+дескрипшн + пикчу ниже)

@EliseyMartynov сделаешь пж? 🙏 

![todo-app-react-effector](https://user-images.githubusercontent.com/42924400/188843320-b0260459-f9bf-47cb-a974-5e0e971948c9.png)

Reply:
@azinit ой, на обеде писал, затупил. Имел в виду _Redux version_. Дублирование типо в названии Redux есть и в описании тоже. Ок с этим нам?
Если да, то также и в эффекторе сделаю.

Reply:
Да, норм)